2020-02-Spring/4_graphs/README.md

```diff
@@ -48,22 +48,78 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. PROBLEM 2
 
-Source: TODO :bug:
 
 #### Scenario
 
-Problem Statement TODO :bug:
-
-#### Example Input
+The electrical system of HappyLand consists of ``N`` stations from ``0`` to ``N-1`` including ``M`` power plants where electricity is produced and the rest ``N-M`` stations are transmission substations, ``M <= N``. After a hurricane, HappyLand’s authorities need to turn off some stations for maintenance. For safety reasons, an offline station has no electricity coming in/out of it. Consequently, substations rely only on offline stations are disconnected from the grid. Power plants can power themselves and adjacent stations. **Find out which stations are shutdown by disconnecting K stations.**
```

Comment:
I feel like there's a grammar mistake in this sentence that makes it confusing:
`Consequently, substations rely only on offline stations are disconnected from the grid.`

Reply:
I too find this confusing

Reply:
thanks guys. pls take a look at it again. let me know if it needs to be changed